bin/aapp_dr_runner.py

```diff
@@ -714,9 +714,9 @@ def generate_process_config(msg, config):
         return False
 
     config['start_time'] = msg.data['start_time']
-
-    # Save collection_area_id if given
-    config['collection_area_id'] = msg.data.get('collection_area_id', None)
+    # Save collection_area_id if given, use static default from config otherwise:
+    default_collection_area_id = config['aapp_processes'][config.process_name]['collection_area_id']
```

Comment:
Wouldn't this fail if an old version (previously working!) config is used? Maybe use `.get(..., None)` for backwards compatibility.

Reply:
@pnuu Even added a unittest. The runner would be helped with some refactoring! There is way too much stuff in the runner script. Now moved a tiny bit to a new module...

Reply:
Ok to merge @pnuu ?
I see that flake8 has a lot of complaints, seems like we have never flak8'ified this repo yet!?
I will leave that for a separate PR, ok?
Will make an issue...

Reply:
I trust your judgment :grin: Fixing the flake8 complaints in a later PR is ok to me.